src/main/java/mongodb/Config.java

```diff
@@ -0,0 +1,6 @@
+package mongodb;
+
+public class Config
+{
+	public final static String URI_STRING = "mongodb+srv://m001-student:m001-mongodb-basics@sandbox.zzlni.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
I accidentally put the full login and password in a public repo, this is not recommended